src/core/utils/templating/helpers/sanitizationHelpers.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Rename this to utils. We'll only keep helpers in this. This isn't a templating helper